tx/textual/valuerenderer/protoreflect_list.go

```diff
@@ -1,4 +1,4 @@
-package valuerenderer_test
+package valuerenderer
```

Comment:
I don't think we should expose this GenericList in the public API. If we need it outside of tests, can we move it to textual's `internal` package?